### ISSUE TITLE
(WCAG 2.1) Applicant program list page button aria-labels are incorrect

### ISSUE THREAD

**Issue by jeanniefu** [civiform/civiform#3552]
Required to meet WCAG 2.1 guidelines. Identified during an accessibility audit against WCAG 2.1.

**WCAG SC**:  [2.5.3 Label in Name](https://www.w3.org/TR/WCAG21/#label-in-name)

**Observed results**: The Edit and Continue buttons all have accessible labels that start with "Apply..."

**How to fix**: Update the `aria-label` attribute text on the Continue and Edit buttons to say "Continue..." and "Edit...".

![Screen Shot 2022-09-28 at 3 29 10 PM](https://user-images.githubusercontent.com/9094240/192899875-0b84b7d6-f04e-4154-8abb-20fa8a533212.png)

**Comment by clouser2** [comment]
This one's a bit tricky since we also localize the screenreader text. As part of creating the new sections, we added localization for the rendered text, but failed to render a screenreader equivalent.

For example, the "Apply" button has screenreader text of "Apply to <program name>".

If an application is in progress, we show "Continue" visually, but don't have a translation for this yet.

If an application has already been submitted, we show "Edit".

**Proposal**

For both submitted / in-progress applications, add new entries for strings to be translated but don't yet consume them from the UI. Once the translations are in, make use of them.

While waiting for those translations, use the existing entry we have for editing a question here for both submitted / in-progress applications ([code](https://github.com/civiform/civiform/blob/9e5d27e8540d04a4fe40cc98c071d759eb0b2221/server/conf/messages#L184-L186)).
